test/system/250-systemd.bats

```diff
@@ -400,4 +400,23 @@ EOF
     run_podman rm -f -t 0 $cname
 }
 
+@test "podman-system-service containers --host" {
+    skip_if_remote "N/A under podman-remote"
+
+    SERVICE_NAME=podman-service-$(random_string)
+    port=$(random_free_port)
+    URL=tcp://127.0.0.1:$port
+
+    systemd-run --unit=$SERVICE_NAME $PODMAN system service $URL --time=0
+    wait_for_port 127.0.0.1 $port
+
+    run_podman --host $URL run --rm $IMAGE true
+    run_podman -H $URL run --rm $IMAGE true
+
+    systemctl stop $SERVICE_NAME
+
+    # Make sure the option is actually connecting
+    run_podman 125 --host $URL run --rm $IMAGE true
+    assert "$output" =~ "Cannot connect to Podman.*connection refused"
```

Comment:
Why do we run containers? Could we just call info? That way we can also check for remote.

Reply:
The goal here is to make sure the --host $URL is actually doing something including failing if the socket does not exist.

Reply:
Yes and info fails without socket.

Reply:
And you think info would be faster?  I just used what was suggested by Ed.

Reply:
I would say yes but it is not really important enough to argue over it. I am ok to merge this.

Reply:
I just made my suggestion because I didn't think `podman info` would show the socket. (There's a bug open for that, or maybe it's been fixed, I don't know). Anyway, if `podman info` shows the socket name, that is a much better solution than running containers.

Reply:
Hey, the `info` bug is fixed. Suggestion:
```bash
@test "podman-system-service containers --host" {
    skip_if_remote "N/A under podman-remote"

    SERVICE_NAME=podman-service-$(random_string)

    # (Used only in teardown if the test fails)
    touch $UNIT_FILE

    port=$(random_free_port)
    URL=tcp://127.0.0.1:$port

    systemd-run --unit=$SERVICE_NAME $PODMAN system service $URL --time=0
    wait_for_port 127.0.0.1 $port

    for opt in --host -H; do
        run_podman $opt $URL info --format '{{.Host.RemoteSocket.Path}}'
        is "$output" "$URL" "RemoteSocket.Path using $opt"
    done

    systemctl stop $SERVICE_NAME
    rm -f $UNIT_FILE
}
```
This also adds an important cleanup step to prevent leaving the podman server behind on failure.

Reply:
And yes, I realize that the previous test (from which this was copied) also has a missing cleanup. That's now on my TODO list to fix. Unless you'd like to fix it.

Reply:
Lets get this merged and then fix the test, unless something else fails.

Reply:
OK, I'll fix it in a cleanup PR later